docs/release-notes/2.17.0.md

```diff
@@ -0,0 +1,74 @@
+
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - DBaaS is now is better open for the public, so you can quickly turn it ON  on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
+    - Database components management will enable PMM administrators to restrict some DB versions from creating DB Clusters.
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+
+
+## New Features
+
+* [PMM-7863](https://jira.percona.com/browse/PMM-7863): Add version management for HAProxy
+* [PMM-7848](https://jira.percona.com/browse/PMM-7848), [PMM-7847](https://jira.percona.com/browse/PMM-7847), [PMM-7421](https://jira.percona.com/browse/PMM-7421): Add support for using SSL certificates between pmm-admin and monitored MySQL databases
+* [PMM-7883](https://jira.percona.com/browse/PMM-7883): Single-line install of PMM Server on supported Linux distros - [Technical Preview]
+* [PMM-7013](https://jira.percona.com/browse/PMM-7013), [PMM-7819](https://jira.percona.com/browse/PMM-7819): Use HAProxy by default instead of ProxySQL
+* [PMM-7356](https://jira.percona.com/browse/PMM-7356), [PMM-7581](https://jira.percona.com/browse/PMM-7581): Management of available versions of components
+* [PMM-7358](https://jira.percona.com/browse/PMM-7358), [PMM-7576](https://jira.percona.com/browse/PMM-7576): Default version for Create DB cluster
+
+## Improvements
+
+* [PMM-7572](https://jira.percona.com/browse/PMM-7572): Add TLS options to mysqld_exporter
+* [PMM-7783](https://jira.percona.com/browse/PMM-7783): Support of pg_stat_monitor v0.9.0
+* [PMM-7064](https://jira.percona.com/browse/PMM-7064): Integrated Alerting: Presenting severity of the Alert Rule using different colors
+* [PMM-7946](https://jira.percona.com/browse/PMM-7946): Better error message on PMM client if server doesn't support HAProxy
+* [PMM-7932](https://jira.percona.com/browse/PMM-7932): Usability improvements on UI for adding External Services
+* [PMM-7641](https://jira.percona.com/browse/PMM-7641), [PMM-7820](https://jira.percona.com/browse/PMM-7820): Make DBAAS feature flag - same as other Feature Flags
+* [PMM-7966](https://jira.percona.com/browse/PMM-7966): Telemetry: Collect enabled/disabled status for Integrated Alerting and Security Threat Tool features
+
+## Bugs Fixed
+
+* [PMM-7911](https://jira.percona.com/browse/PMM-7911): DBaaS: Invalid Number of Nodes results in bad request
+* [PMM-7884](https://jira.percona.com/browse/PMM-7884): DBaaS: Fix DB Cluster tab loading
+* [PMM-7941](https://jira.percona.com/browse/PMM-7941): mongodb_exporter provides a wrong replication status value
```

Comment:
actually this one didn't get into the release. We found out just recently that it is not included.

Reply:
PMM-7941 was removed